cmd/tnf/claim/show/failures/failures.go

```diff
@@ -265,6 +270,25 @@ func getFailedTestCasesByTestSuite(claimResultsByTestSuite map[string][]*claim.T
 	return testSuites, nil
 }
 
+func checkClaimVersion(version string) error {
+	claimSemVersion, err := semver.NewVersion(version)
+	if err != nil {
+		return fmt.Errorf("claim file version %q is not valid: %v", version, err)
+	}
+
+	supportedSemVersion, err := semver.NewVersion(supportedClaimFormatVersion)
+	if err != nil {
+		return fmt.Errorf("supported claim file version v%v is not valid: v%v", supportedClaimFormatVersion, err)
+	}
+
+	if claimSemVersion.Compare(supportedSemVersion) < 0 {
```

Comment:
Shouldn't we check for equality here? Otherwise, we will have to support old format versions which might be complex. This would still support any number of TNF suite exes supporting this format version.

Reply:
Done in https://github.com/test-network-function/cnf-certification-test/pull/1258/commits/c650ea25f7c7b7057c91119065aa853478f53f35